CHANGELOG.md

```diff
@@ -302,6 +302,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Removed deprecated trainer attributes - `get_model` and `accelerator_backend` ([#7502](https://github.com/PyTorchLightning/pytorch-lightning/pull/7502))
 
 
+- Removed support for automatically monitoring the `val_loss` key with `ModelCheckpoint` ([#8293](https://github.com/PyTorchLightning/pytorch-lightning/pull/8293))
+
```

Comment:
Let's also add a separate entry under changed, notifying about monitor becoming mandatory?
Or include here in this message?

Reply:
Updated